### PR TITLE
Email_validation

### PR DESCRIPTION
**Pull request by AntonioHincapie**
-Added message error.
-Added the script for this message.
-Finished email validation.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -91,4 +91,26 @@ for (let i = 0; i < buttonsProject.length; i += 1) {
...
+const emailForm = document.getElementById('mail');
+const errorMessage = document.getElementById('error-message');
+
+const emailLowercase = (mail) => {
+  const itLowerc = new RegExp(/[A-Z]/g);
+  if (itLowerc.test(mail)) {
+    return false;
+  }
+  return true;
+};
```
Comment (CollinsTatang) [inline script.js:106]: - [x] Please you have to check on submitting the form if the email is all lowercase if not it should return a friendly message telling the user to input their email address all in lowercase.

File: script.js [hunk 2]
```diff
@@ -91,4 +91,26 @@ for (let i = 0; i < buttonsProject.length; i += 1) {
...
+
+contForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (emailLowercase(emailForm.value.trim())) {
+    contForm.submit();
+    errorMessage.style.display = 'none';
+  } else {
+    errorMessage.style.display = 'block';
+  }
```
Comment (CollinsTatang) [inline script.js:115]: - [x] Your error message doesn't display when a wrong email is input and submit

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (AntonioHincapie) [reply script.js:106]: image
Hi @CollinsTatang the image below shows the error message when you put some upper case to the email input, please clarify your feedback to improve my code.

File: script.js (hunk 2 above)
Comment (AntonioHincapie) [reply script.js:115]: image
Hi @CollinsTatang  the image below shows the error message when you put your email wrong to the email input, please clarify your feedback to improve my code.

Comment (CollinsTatang) [reply script.js:115]: Great job done, it works fine now

File: script.js (hunk 1 above)
Comment (CollinsTatang) [reply script.js:106]: Great job done, everything Ok now

**Review by codepantha — APPROVED** [review]
Hi @AntonioHincapie 👋, 

## STATUS: APPROVED ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**